lib/utils.js

```diff
@@ -30,4 +32,16 @@ const getAgentInfo = () => ({
   name: pjson.name,
 });
 
-module.exports = { getCodeRef, getAgentInfo };
+const parseStringToArray = (str) => {
+  if (typeof str === typeof TYPE_STRING) {
+    return str.split(';').map((attribute) => {
+      const keyAndValue = attribute.split(':');
+      return keyAndValue.length === 2 && typeof keyAndValue !== typeof TYPE_STRING
```

Comment:
When is the case with `keyAndValue` string possible? Seems like it will be array in any case.